tank_battalion/tank_battalion.pde

```diff
@@ -69,13 +69,26 @@ void keyReleased() {
 void draw() {
   background(0);
   
+  //Draw background
+  imageMode(CORNER);
+  image(background_image, 0, 0, width, height);
+  
```

Comment:
I see you've changed this back to be the first image to be drawn, but I conciously moved it. If this is the first image to be drawn, you get this black line:
![black_line](https://user-images.githubusercontent.com/44906912/92788497-48d30400-f3aa-11ea-9697-a4290dc3f107.png)
So I suggest t oeither move it back to be the last image, or atleast move it past the grid.draw()

Reply:
Yes I noticed it as well. Its only a bit weird drawing the background after the other objects.